package.json

```
@@ -78,15 +83,16 @@
     "start": "react-native start",
     "start:android": "react-native run-android",
     "start:ios": "react-native run-ios",
-    "lint": "eslint index.js --max-warnings=0",
+    "lint": "eslint index.tsx",
     "lint:staged": "lint-staged",
     "release:major": "node ./changelog --major && npm version major && git push origin && git push origin --follow-tags",
     "release:minor": "node ./changelog --minor && npm version minor && git push origin && git push origin --follow-tags",
     "release:patch": "node ./changelog --patch && npm version patch && git push origin && git push origin --follow-tags",
     "release:version": "node ./changelog --version $VERSION && git push origin && git push origin --follow-tags",
-    "validate": "yarn validate:prettier && yarn validate:lint && yarn validate:flow",
-    "validate:flow": "yarn flow",
-    "validate:lint": "eslint *js components/*.js --max-warnings=0",
+    "validate": "yarn validate:prettier && yarn validate:lint && yarn validate:flow && validate:ts",
```

I think you can remove flow validation

```suggestion
    "validate": "yarn validate:prettier && yarn validate:lint && yarn  validate:ts",
```